docs/GettingStarted.md

```diff
@@ -186,19 +186,19 @@ executions. It can be added as any other Sidekiq middleware:
 
     Sidekiq.configure_server do |config|
       config.server_middleware do |chain|
-        chain.add(Datadog::Contrib::Sidekiq::Tracer, debug: true)
+        chain.add(Datadog::Contrib::Sidekiq::Tracer)
       end
     end
 
-#### Configure the tracer
+#### Configure the tracer middleware
 
 To modify the default configuration, simply pass arguments to the middleware.
 For example, to change the default service name and activate the debug mode:
 
     Sidekiq.configure_server do |config|
       config.server_middleware do |chain|
         chain.add(Datadog::Contrib::Sidekiq::Tracer,
-                  default_service: 'my_app', debug: true)
+                  sidekiq_service: 'my_app', debug: true)
```

Comment:
Let's remove these `debug: true` from the example snippets.
And also give a nicer name to this customized service name (example: `sidekiq-notifications`)